libs/langchain/langchain/llms/edenai.py

```diff
@@ -131,10 +137,12 @@ def _call(
             "providers": self.provider,
             "num_images": 1,  # always limit to 1 (ignored for text)
             "text": prompt,
-            **kwargs,
```

Comment:
i think it's probably fine to keep passing these in (and would be a breaking change to remove them